File: src/main/java/kernitus/plugin/hotels/bukkit/Utilities.java
package kernitus.plugin.hotels.bukkit;

import org.bukkit.Bukkit;
import org.bukkit.World;

import java.util.Optional;
import java.util.UUID;

public class Utilities {

    public static Optional<UUID> worldNameToId(String name) {
        return Optional.ofNullable(Bukkit.getWorld(name)).map(World::getUID);
    }
}

File: src/main/java/kernitus/plugin/hotels/bukkit/Utilities.kt
package kernitus.plugin.hotels.bukkit

import org.bukkit.Bukkit
import org.bukkit.World
import java.util.*

object Utilities {

    fun worldNameToId(name: String): Optional<UUID> {
        return Optional.ofNullable(Bukkit.getWorld(name)).map(Function<T, UUID> { World.getUID() })
    }
}
